<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegionAdvisorWidget</class>
 <widget class="QWidget" name="LegionAdvisorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Button" name="cIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label" name="cTitle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>12</y>
     <width>545</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="MessageWidget" name="cLegionList" native="true">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>70</y>
     <width>576</width>
     <height>272</height>
    </rect>
   </property>
  </widget>
  <widget class="Button" name="cReportsButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>359</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cRequestButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>379</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label" name="cMorale">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>58</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Morale</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Label" name="cGoto">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>43</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Label" name="cLegion">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>58</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Legion</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Label" name="cReturn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>43</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Label" name="cToFort">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>58</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>to fort</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Label" name="cEmpire">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>43</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Empire</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Label" name="cService">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>58</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>service</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Label" name="cReports">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>358</y>
     <width>500</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reports</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Label" name="cRequests">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>378</y>
     <width>500</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Requests</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Label" name="cNoLegions">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>200</y>
     <width>496</width>
     <height>63</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>widget/button.h</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>widget/label.h</header>
  </customwidget>
  <customwidget>
   <class>MessageWidget</class>
   <extends>QWidget</extends>
   <header>widget/messagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
